docs/usage/loading-packages.md

```diff
@@ -5,81 +5,62 @@
 
-Packages included in the official Pyodide repository can be loaded using
-{any}`pyodide.loadPackage`:
+While {any}`micropip.install` is written in Python and
+{any}`pyodide.loadPackage` in Javascript this has no incidence on when
+to use each of these functions. Indeed, you can easily switch languages using
+the {ref}`type-translations` with,
 
-```js
-await pyodide.loadPackage("numpy");
-```
+- from Javascript,
+  ```javascript
+  let micropip = pyodide.pyimport(package_name);
+  ```
+- from Python,
+  ```
+  import pyodide_js
+  await pyodide_js.loadPackage('package_name')
+  ```
 
-It is also possible to load packages from custom URLs:
+Instead, the general advice is to use {any}`micropip.install` for everything
+except in the following cases where {any}`pyodide.loadPackage` might be more
+appropriate,
 
-```js
-await pyodide.loadPackage(
-  "https://foo/bar/numpy-1.22.3-cp310-cp310-emscripten_3_1_13_wasm32.whl"
-);
-```
-
-The file name must be a valid wheel name.
-
-When you request a package from the official repository, all of the package's
-dependencies are also loaded. Dependency resolution is not yet implemented when
-loading packages from custom URLs.
-
-In general, loading a package twice is not permitted. However, one can override
-a dependency by loading a custom URL with the same package name before loading
-the dependent package.
-
-Multiple packages can also be loaded at the same time by passing a list to
-{any}`pyodide.loadPackage`.
-
-```js
-await pyodide.loadPackage(["cycler", "pytz"]);
-```
-
-{any}`pyodide.loadPackage` returns a `Promise` which resolves when all the
-packages are finished loading:
-
-```javascript
-let pyodide;
-async function main() {
-  pyodide = await loadPyodide();
-  await pyodide.loadPackage("matplotlib");
-  // matplotlib is now available
-}
-main();
-```
-
-(micropip)=
+- to load micropip itself,
+- when you are optimizing for size, do not want to install the `micropip`
+  package, and do not need to install packages from PyPI.
 
 ## Micropip
 
-### Installing packages from PyPI
+### Installing packages
+
+Pyodide supports installing following types of packages with {mod}`micropip`,
 
-Pyodide supports installing pure Python wheels from PyPI with {mod}`micropip`.
-{func}`micropip.install` returns a Python
-[Future](https://docs.python.org/3/library/asyncio-future.html) so you can await
-the future or otherwise use the Python future API to do work once the packages
-have finished loading:
+- pure Python wheels from PyPI with {mod}`micropip`.
+- pure Python and binary wasm32/emscripten wheels (also informally known as
+  "Pyodide packages" or "packages built by Pyodide") from the JsDelivr CDN and
+  custom URLs.
+  {func}`micropip.install` returns a Python
+  [Future](https://docs.python.org/3/library/asyncio-future.html) so you can
+  await the future or otherwise use the Python future API to do work once the
+  packages have finished loading:
```

Comment:
This is out of date, currently `micropip.install` is a normal `async` method so it returns a coroutine. So if you **don't** either await it or schedule it with `asyncio.ensure_future` it won't happen and you'll probably get a message like ` RuntimeWarning: coroutine 'f' was never awaited`. This was a breaking change in v0.20.0 but it seems that most people were `await`ing the result so nobody has complained.

Reply:
OK thanks, I'll just change to say it's a async Python that returns a coroutine. People shouldn't be using ensure_future for this.